wondrous-app/components/Common/CSVFileDropZone/styles.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import palette from 'theme/palette';
+
+export const DropZone = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 40px 0;
+  border: 2px dashed ${palette.grey78};
+  border-radius: 6px;
+  cursor: pointer;
+  color: ${palette.grey78};
+  font-size: 14px;
+  font-weight: 500;
+  transition: all 0.2s ease-out;
+
+  :hover {
```

Comment:
This is a difference. 

![screen-2022-10-18-21-53-07-2v665](https://user-images.githubusercontent.com/1454782/196550129-d87c641c-0a90-43ee-9061-2fae6610c16e.jpg)

Reply:
gotcha, thanks for the explanation :raised_hands: